### DISCUSSION TITLE
How to Configure Separate Log Files for Access and Service Logs in a Spring Project Using Logback and Logbook?

### DISCUSSION THREAD

**Discussion by chungyeong** [zalando/logbook#1881] [Q&A]
My Spring project is using Logback and Logbook. I want to log access and service separately and create log files project.access.log and project.service.log. Is there a way to configure this?

**Comment by whiskeysierra** [comment]
That's a logback question. You need to configure logback to log certain log categories/names to certain files.
Logbook uses `org.zalando.logbook.Logbook` by default (see https://github.com/zalando/logbook?tab=readme-ov-file#writing).

This might help: https://stackoverflow.com/questions/2488558/logback-to-log-different-messages-to-two-files
